tools/docker/quorum-multi-party-all-in-one/README.md

```diff
@@ -0,0 +1,106 @@
+# @hyperledger/cactus-quorum-multi-party-all-in-one<!-- omit in toc -->
+
+## Table of Contents<!-- omit in toc -->
+
+- [Summary](#summary)
+- [Usage via Public Container Registry](#usage-via-public-container-registry)
+- [List endpoints and services](#list-endpoints-and-services)
+- [2021-06-16 22:23:46,653 DEBG 'quorum-network' stdout output:](#2021-06-16-222346653-debg-quorum-network-stdout-output)
+- [List endpoints and services](#list-endpoints-and-services-1)
+
+## Summary
+
+A container image that hosts a Quorum network which is
+- Has multiple nodes and validators
+- Supports transaction privacy (`privateFrom` and `privateFor`)
+
+## Usage via Public Container Registry
+
+```sh
+docker run \
+  --rm \
+  --privileged \
+  --publish 2222:22 \
+  --publish 3000:3000 \
+  --publish 8545:8545 \
+  --publish 8546:8546 \
+  --publish 9001:9001 \
+  --publish 9081:9081 \
+  --publish 9082:9082 \
+  --publish 9083:9083 \
+  --publish 9090:9090 \
+  --publish 18545:18545 \
+  --publish 20000:20000 \
+  --publish 20001:20001 \
+  --publish 20002:20002 \
+  --publish 20003:20003 \
+  --publish 20004:20004 \
+  --publish 20005:20005 \
+  --publish 25000:25000 \
+  petermetz/cactus-quorum-multi-party-all-in-one:0.1.0
```

Comment:
@travis-payne Please confirm if this command works on the terminal without having to build an image locally.